protocol/tests/regtest.sh

```diff
@@ -0,0 +1,3 @@
+bitcoind --chain=regtest --txindex --blockfilterindex --peerblockfilters --rpcport=18443 --rpcuser=test --rpcpassword=b324 --rest=1 --server=1 --listen=1 --v2transport=1 &
+sleep 1
```

Comment:
Would be pretty easy to add a timeout to this in the future, maybe when we get it in to CI.